README.md

```diff
@@ -174,34 +172,38 @@ Or add the path to the `bin` folder to your `$PATH` and invoke `swiftgen` direct
 
 SwiftGen is provided as a single command-line tool which uses a configuration file to run various actions (subcommands).
 
+To create a sample configuration file as a starting point to adapt to your needs, run `swiftgen config init`.
+
 Each action described in the [configuration file](Documentation/ConfigFile.md) (`strings`, `fonts`, `ib`, …) typically corresponds to a type of input resources to parse (strings files, IB files, Font files, JSON files, …), allowing you to generate constants for each types of those input files.
 
-To use SwiftGen, simply create a `swiftgen.yml` YAML file to list all the subcommands to invoke, and for each subcommand, the list of arguments to pass to it. For example:
+To use SwiftGen, simply create a `swiftgen.yml` YAML file (either manually or using `swiftgen config init`) then edit it to adapt to your project. The config file should list all the subcommands to invoke, and for each subcommand, the list of inputs/outputs/templates/parameters to use for it.
+
+For example:
 
 ```yaml
 strings:
   inputs: Resources/Base.lproj
-  filter: .+\.strings$
```

Comment:
I thought that this `filter` was more confusing than anything in the README, as it's an advanced feature. It's documented in the `ConfigFile.md` detailed documentation, so better not lose the newcomers with that in README.